tiledb/sm/array_schema/array_schema_operations.h

```diff
@@ -76,6 +79,16 @@ void store_array_schema(
     const shared_ptr<ArraySchema>& array_schema,
     const EncryptionKey& encryption_key);
 
+/**
+ * Handles loading an array schema from a URI.
+ *
+ * @param ctx TileDB Context.
+ * @param uri The URI of the array to load schema.
+ * @param config TileDB Config. If null, the context config will be used.
+ */
```

Comment:
@bekadavis9 do you think this belongs here? I'm not sure we want to add the dependency to array directory if we don't have to.

Reply:
Yes, this is fine. This file is intended to separate out the I/O-based `ArraySchema` operations and will undoubtedly rely on `ArrayDirectory` at some point; it just hasn't needed to yet. The `array` CAPI Handle will already need to link against `capi_array_schema_stub` to make use of its internals, so there's no additional overhead here that I can think of. It could be argued that it's a potentially unnecessary change, and the body could stay as-is in the C API, but I have no real qualms about the change.